build-arch-gce

```diff
@@ -76,23 +78,11 @@ append_gce_repo() {
 cp /etc/pacman.conf "$work_dir"
 append_gce_repo "$work_dir/pacman.conf"
 pacstrap -G -M -C "$work_dir/pacman.conf" -- "$mount_dir" \
-	base linux grub dosfstools e2fsprogs dhclient openssh sudo google-compute-engine growpartfs
+	base linux dosfstools e2fsprogs dhclient openssh sudo google-compute-engine growpartfs
 append_gce_repo "$mount_dir/etc/pacman.conf"
 
 echo '- Configuring fstab.'
-root_uuid=$(lsblk --noheadings --raw --output UUID -- "$root_dev")
-efi_uuid=$(lsblk --noheadings --raw --output UUID -- "$efi_dev")
-print_fstab() {
-	printf '# LABEL=%s\n' "$1"
-	printf 'UUID=%-20s' "$2"
-	printf '\t%-10s' "$3" "$4" "$5"
-	printf '\t%s %s' "$6" "$7"
-	printf '\n\n'
-} >> "$mount_dir/etc/fstab"
-{
-	print_fstab root "$root_uuid" / ext4 rw,discard,errors=remount-ro 0 1
-	print_fstab efi "$efi_uuid" /efi vfat uid=root,gid=root,umask=022,showexec 0 0
-}
+cp -f -- ./sys/etc/fstab $mount_dir/etc/fstab
```

Comment:
This makes the script not self contained, which is going to break this package: https://aur.archlinux.org/packages/build-arch-gce

Reply:
I see, got it.